android/gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,5 +1,6 @@
-distributionBase=GRADLE_USER_HOME
-distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-7.4.2-all.zip
-zipStoreBase=GRADLE_USER_HOME
-zipStorePath=wrapper/dists
+#Wed Aug 03 15:32:22 IST 2022
+distributionBase=GRADLE_USER_HOME
+distributionUrl=https\://services.gradle.org/distributions/gradle-7.3.3-bin.zip
+distributionPath=wrapper/dists
+zipStorePath=wrapper/dists
+zipStoreBase=GRADLE_USER_HOME

```

Comment:
@Charan017 please fix the ^M (Ctrl + M) as new lines as per GitHub recommendation using this solution: https://stackoverflow.com/a/1889699